### PR TITLE
resolve warnings on Manage Jenkins screen and update plugins

### PR DESCRIPTION
**Pull request by mdelgin**

### CODE REVIEW

File: cjt/jenkins.yaml [hunk 1]
```diff
@@ -1,6 +1,30 @@
...
+        - id: ${JENKINS_USER}
+          password: ${JENKINS_PASSWORD}
+  remotingSecurity:
+    enabled: true
+  slaveAgentPort: 50000
+security:
+  remotingCLI:
+    enabled: false
+  sshd:
+    port: 50001
```
Comment (schottsfired) [inline cjt/jenkins.yaml]: personally I'm a fan of 2222

File: cjt/jenkins.yaml [hunk 2]
```diff
@@ -11,7 +35,7 @@ unclassified:
               git:
                 remote: "https://github.com/cloudy-demos/pipeline-libraries.git"
 jobs:
-  - >
+  - script : >
```
Comment (schottsfired) [inline cjt/jenkins.yaml:38]: what is this for?

File: cjt/plugins.txt [hunk 3]
```diff
@@ -112,7 +112,8 @@ cloudbees-support:3.18
 favorite:2.3.1
 blueocean-bitbucket-pipeline:1.5.1
 mapdb-api:1.0.9.0
-configuration-as-code:0.9-alpha
+configuration-as-code:1.0-rc1
```
Comment (schottsfired) [inline cjt/plugins.txt:115]: sweet!

**Review by schottsfired — APPROVED** [review]

File: cjt/jenkins.yaml (hunk 2 above)
Comment (mdelgin) [reply cjt/jenkins.yaml:38]: the newer configuration-as-code version has slightly different syntax for job configuration

**Comment by schottsfired** [comment]
Tested this and everything works. No more ugly warnings OOTB \o/

**Pull request merged**